app/models/user.rb

```diff
@@ -0,0 +1,24 @@
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
+  def is_post_owner?(post)
+    posts.include?(post)
+  end
+
+  def is_likes_owner?(like)
```

Comment:
Style/PredicateName: Rename `is_likes_owner?` to `likes_owner?`. (https://github.com/bbatsov/ruby-style-guide#bool-methods-qmark)